pkg/apis/policy/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -161,6 +161,9 @@ type TLog struct {
 	// URL sets the url to the rekor instance (by default the public rekor.sigstore.dev)
 	// +optional
 	URL *apis.URL `json:"url,omitempty"`
+	// Use the Public Key from the referred TrustRoot.TLog
+	// +optional
+	TrustRootRef string `json:"trustRootRef,omitempty"`
```

Comment:
I assume this value is referred to the name of the TrustRoot, don't you prefer to use an ObjectReference instead ?

Reply:
Felt like overkill here since then we'd need to plumb in the gvk stuff as well. If you feel strongly, I can change it, just felt it would convolute the API unnecessarily.

Reply:
Definitely it isn't a blocker. You could use a LocalObjectReference due to the cluster scope of trustroots.